src/runtime/virtcontainers/hypervisor.go

```diff
@@ -238,312 +238,103 @@ type Param struct {
 
 // HypervisorConfig is the hypervisor configuration.
 type HypervisorConfig struct {
-	// customAssets is a map of assets.
```

Comment:
@fidencio @jimcadden @fitzthum @dubek As part of the plan to merge CCv0 and main, I wonder if we want to keep the change done here that removed all the comments. This is still visible in CCv0 vs main today.

What was the rationale for it? Should we fixup that part and restore the comments to be in line with main?

Reply:
I'm working on the PR to incorporate these changes in main. I'll leave the comments in place in main.